app/src/main/res/raw/third_party_licenses

```diff
@@ -0,0 +1,1201 @@
+https://developer.android.com/studio/terms.html
```

Comment:
Yeah. I didn't realize so many dup links are generated automatically...
Thx for your good feedback!

Reply:
hmm... if its generated i don't know if we should fix or not. because the next time we generate it (if we add a new library) they will be added again

Reply:
@panpanini 
I just checked out the history of this file, and all of them are completely generated automatically.
I might think we shouldn't change some of these manually. 
What do you say?

Reply:
Your comments are faster. 

> because the next time we generate it (if we add a new library) they will be added again

agree. 
Let me try more. I'll try to add new one.

Reply:
looks like that. 

https://developers.google.com/android/guides/opensource

> Each AAR or JAR for a library contains a POM file which describes the contents of the archive. The gradle plugin processes the Licenses field in this file, if present.

maybe still under ver 1.0? 
They don't filter duplications maybe? 

Hmm. how can we handle this.

Reply:
I think because its generated, we should leave it. (We don't know how the OssFragment is implemented, so we shouldn't change the data that they are expecting.) so this is LGTM for me! 💪

Reply:
(The Fragment itself appears to work correctly, and there is no strange data being displayed from what I could see)